spec/features/boats/show_spec.rb

```diff
@@ -13,6 +13,19 @@
     expect(page).to have_content(@hobie.name)
     expect(page).to have_content(@hobie.motor_powered)
     expect(page).to have_content(@hobie.crew_size)
-    save_and_open_page
+  end
+
+  scenario "visitor clicks link and go to boat index" do
+    visit "/boats/#{@hobie.id}"
+
+    click_link('All Boats')
+    expect(page).to have_current_path('/boats')
+  end
+
+  scenario "visitor clicks link and go to dock index" do
+    visit "/boats/#{@hobie.id}"
+
+    click_link('All Docks')
+    expect(page).to have_current_path('/docks')
```

Comment:
I didn't notice this before so I'm not sure if it was on the other pages too, but you can do expect(current_path).to eq